### ISSUE TITLE
iOS crash after taking picture

### ISSUE THREAD

**Issue by yesprasoon** [cordova-plugin-camera-preview/cordova-plugin-camera-preview#395]
This happens with the sample app, as well as, if I add this plug-in to my project.

After taking picture, the whole app gets crashed, with following error in alert box in Xcode.

> Restore the connection to “XXXX iPad” and run “cordova-plugin-camera-sample-app” again, or if “cordova-plugin-camera-sample-app” is still running, you can attach to it by selecting Debug > Attach to Process > cordova-plugin-camera-sample-app.

This is very unpredictable as sometime it happens after taking 5-6 images, sometime after 15-20.

I am making following changes in startCamera options:
`x: 0, y: 0, width: window.innerWidth, height:window.innerHeight`
Even if I do not do this change, it crashes in my application after sometime but not in Sample app.

My iOS device - iPad Mini, v9.3.5, Capacity 12.7GB, Available 7.0GB

**I always make sure that I do not takePicture unless I get success result from previous request.**

**Comment by lec00q** [comment]
Hello @yesprasoon ! I suspect this is _exactly_ the same as my issue #393. iPad Mini has the same hardware configuration as an iPhone 6/6Plus.
At least I am glad I am not alone 🙂 

This is a severe bug IMHO, I really hope that someone can help us because it is affecting our application in production...

**Comment by abanoubsamaan** [comment]
the same for me Iphone 5s

**Comment by ngt14** [comment]
+1

**Comment by Choppel** [comment]
@yesprasoon @lec00q 
It looks like the sample app pulls the plugin from github (master). There are known issues with this version (see #401 and others). Please try the one from npm instead (0.9.0).
```
<plugin name="cordova-plugin-camera-preview" spec="^0.9.0" />
```
Also, please don't use the sample app. It may be outdated (last commit from march 2017).

**Comment by ryaa** [comment]
This problem could be related to https://github.com/cordova-plugin-camera-preview/cordova-plugin-camera-preview/issues/604

**Comment by westonganger** [comment]
Closed via #604
